app/settings.py

```diff
@@ -27,6 +27,13 @@ def get_key(_key_name):
     else:
         return None
 
+
+def get_application_version_from_file():
```

Comment:
I think there's already a function below that does this (although it might be key related) possible refactor to use a common one?